### DISCUSSION TITLE
Icon request: icon-sql

### DISCUSSION THREAD

**Discussion by ps3331333** [FortAwesome/Font-Awesome#21172] [Icon Requests]
The icon will have the word "SQL" . A user will click on the icon/button and the application will show the SQL syntax.

**Comment by kanalumaddela** [comment]
Is this supposed to be company specific? There's already fa-database which in itself makes sense to most people as SQL. If you want it to show SQL syntax, that would have to be done with html/css.

**Comment by alsoknownasdrew** [comment]
+1

**Comment by SuperPat45** [comment]
+1

**Comment by weinerjm14** [comment]
There are many types of databases. Granted SQL is the most known. But there are several flavors of SQL as well. As a programmer, this would be nice to display on a portfolio site...

**Comment by thystonius** [comment]
I think there **is** president for having a generic "SQL" icon.
There are icons for python and java and JS, and while those have more standardized syntaxes there are significant runtime differences that are similar to SQL.
The "JS" icon may indicate Javascript syntax, but I doesn't mean I can copy that code and just run it without knowing the proper environment.  The fact that something is Javascript does not imply the runtime, flavor, or version (browser vs NodeJS, or ES5 vs ES6, which module syntax, etc.).
Similarly a SQL icon would indicate the general language, but does not imply the flavor or expected runtime (TSQL, PLSQL, Sqlite, etc.).

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
